assets/assets_test.go

```diff
@@ -9,10 +9,12 @@ import (
 
 // TestEmbeddedDocs makes sure we don't forget to regenerate after documentation change
 func TestEmbeddedDocs(t *testing.T) {
+	t.Skip("skipping for now, paths are hard")
```

Comment:
I'm not really familiar with `assets/`: what is this test supposed to do and why are we now skipping it?